lib/parslet/context.rb

```diff
@@ -32,4 +24,31 @@ def initialize(bindings)
       instance_variable_set("@#{key}", value)
     end
   end
+
+  # following methods are needed and were provided by blankslate
```

Comment:
Defining this method just satisfies the specs and serves no purpose. The main challenge in replacing BlankSlate is to get all these methods working, and by working I mean: 

* Can be called
* Show up in respond_to? (both variants)
* Show up in methods
* Both methods and respond_to work as with any other object

This is a show-stopper.